.github/pull_request_template.md

```diff
@@ -1,9 +1,10 @@
-Close #<insert_issue_number>
+Closes #<insert_issue_number>
```

Comment:
i would delete this line altogether. fine to only include in the PR title now and not repeat it twice